operator/api/v1alpha1/securedcluster_types.go

```diff
@@ -317,6 +317,22 @@ type CollectorContainerSpec struct {
 
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=4
 	ContainerSpec `json:",inline"`
+
+	RuntimeConfig *CollectorRuntimeConfig `json:"runtimeConfig,omitempty"`
```

Comment:
This is in `CollectorContainerSpec`, which doesn't seem right since this is in a ConfigMap, not the collector container spec.

Reply:
I think it's fine. The config map still configures the collector which runs inside this container.

However this field should probably go before the deprecated+hidden `ForceCollection`. Please also add comments and kubebuilder/operator-sdk tags here and below as appropriate until this looks friendly and consistent in the OpenShift operator UI (see [deployment instructions](https://github.com/stackrox/stackrox/blob/master/operator/README.md#installing-operator-via-olm)).

Reply:
I have added the tags. I didn't add any tags related to the UI, because I am not planning on having the runtime configuration in the UI.

Reply:
I just want to make sure you don't confuse the stackrox/ACS central UI with the OpenShift operator UI.
Any field that you add to these structs automatically appears in the latter, so appropriate comments and/or tags are not optional here.

Reply:
It would be great if you could also attach some screenshots of how the new elements look like in the operator UI. There were [issues in the past](https://issues.redhat.com/browse/ROX-25872) when changes did not propagate as expected.